.ci/wercker.sh

```diff
@@ -75,9 +75,10 @@ no-error-orekit)
   # no CI is enforced in project, so to make our build stable we should
   # checkout to latest release/development (annotated tag or hash) or sha that have fix we need
   # git checkout $(git describe --abbrev=0 --tags)
-  git checkout "a7e67ce73803c67a""ad90e0b28ed77a7781dc28a9"
+  git checkout "b67b419db7014f4b""ad921a1bac6c848384ad2b92"
```

Comment:
There is spelling false positive in Travis, to fix it we need split this Sha one more time, `bac` should be in different string values

Reply:
Oh, I see! I was wondering why the strings were broken up. I'll fix this right away.